crates/sovereign-sdk/full-node/sov-stf-runner/src/config.rs

```diff
@@ -96,6 +96,8 @@ pub struct FullNodeConfig<DaServiceConfig> {
     pub da: DaServiceConfig,
     /// Important pubkeys
     pub public_keys: RollupPublicKeys,
+    /// Number of blocks to request during sync
+    pub sync_blocks_count: Option<u64>,
```

Comment:
we can make this u64 and default to the other number by a serde macro.

cleaner approach imo. what do you think?